core/src/streams.rs

```diff
@@ -968,11 +968,43 @@ impl<'gc> NetStream<'gc> {
             (_, _, FlvVideoPacket::CommandFrame(_command)) => {
                 tracing::warn!("Stub: FLV command frame processing")
             }
-            (_, _, FlvVideoPacket::AvcSequenceHeader(_data)) => {
-                tracing::warn!("Stub: FLV AVC/H.264 Sequence Header processing")
+            (Some(video_handle), _, FlvVideoPacket::AvcSequenceHeader(data)) => {
+                match context
+                    .video
+                    .configure_video_stream_decoder(video_handle, data)
+                {
+                    Ok(_) => {}
+                    Err(e) => {
+                        tracing::error!("Configuring video decoder {} failed: {}", frame_id, e);
+                    }
+                }
             }
-            (_, _, FlvVideoPacket::AvcNalu { .. }) => {
-                tracing::warn!("Stub: FLV AVC/H.264 NALU processing")
+            (
+                Some(video_handle),
+                Some(codec),
+                FlvVideoPacket::AvcNalu {
+                    composition_time_offset: _,
```

Comment:
I'm going to assume this means Flash didn't actually use the composition time offset in FLV H.264 streams either... AFAIK that sounds plausible to me (FLV is already variable framerate) but I wanted to just double-check.

Reply:
> I'm going to assume this means Flash didn't actually use the composition time offset in FLV H.264 streams either...

Oh, don't assume that! This may be exactly what I asked about in Discord. I just haven't connected the dots...
Also at the beginning, didn't care about anything else than what's strictly necessary for "get picture on screen", and just never got back to this after

Reply:
https://github.com/ruffle-rs/ruffle/pull/15747 should help investigating this.